### DISCUSSION TITLE
Github Actions unresponsive while splitting on push/PR action

### DISCUSSION THREAD

**Discussion by AndrejMitrovic** [community/community#25053] [Actions]
<p>Hi,</p>
<p>In this PR: <a href="https://github.com/bpfkorea/agora/pull/489" rel="nofollow noopener">https://github.com/bpfkorea/agora/pull/489</a> we’ve tried to split our testing so certain compilers are only tested when a PR is made, and another set of compilers are tested when there is a push to our main branch.</p>
<p>However, the Github Actions system for this PR just became unresponsive. There are three runners (ignore the Travis one, it’s unrelated):</p>
<p>Linux (ubuntu-latest, dmd-2.090.0)</p>
<p>Linux (ubuntu-latest, ldc-1.19.0)</p>
<p>MacOS (macOS-latest, dmd-2.090.0)</p>
<p>And it says [Expected — Waiting for status to be reported].</p>
<p>We don’t really know how to debug this. There is no console or log to see what went wrong with the .github.yml script. How do we debug this?</p>
<p>Cheers!</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/andrejmitrovic">@andrejmitrovic</a> ,</p>
<p>I forked your repositories and repro on my side. This is due to the syntax error in your <a href="https://github.com/AndrejMitrovic/agora/blob/7b28cc64cfad0b8b0ddc560290657af8ba6c8089/.github/workflows/github.yml#L60" rel="nofollow noopener">yaml</a> file. You cannot just move the ‘push’ code down to split ‘push/pull_request’. Duplicated workflow in one metadata file is not supported. Please create a new workflow file for ‘push’ event.</p>
<p><img src="https://user-images.githubusercontent.com/52772761/181088582-a9c6da78-cecc-4067-9685-db2eaa8130b2.png" alt="errorcode.png" data-base62-sha1="q7vFOdsMwZS9KdoPfXvXkz9oGQt" width="556" height="396"></p>
<p>I just removed the ‘push’ code part, and workflow starts then.</p>
<p><img src="https://user-images.githubusercontent.com/52772761/181088583-1e11d383-5271-4fc0-8efb-f1fb0ec43344.png" alt="Checkruninpullrequest.png" data-base62-sha1="pCMkvt8sCsZ69FQ23wdQMIt2y6x" width="677" height="281"></p>
